grails-gradle-plugin/src/main/groovy/org/grails/gradle/plugin/core/GrailsGradlePlugin.groovy

```diff
@@ -432,13 +430,6 @@ class GrailsGradlePlugin extends GroovyPlugin {
                     }.join(' ')
                 }
             }
-
-            // assetCompile will be execute for all Jar-type tasks
-            // (see https://github.com/bertramdev/asset-pipeline/blob/master/asset-pipeline-gradle/src/main/groovy/asset/pipeline/gradle/AssetPipelinePlugin.groovy#L85)
-            // exclude the assets from pathing jar
-            task.exclude {
-                it.file.absolutePath.startsWith(assets.absolutePath)
-            }
         }
```

Comment:
Why this workaround for https://github.com/bertramdev/grails-asset-pipeline/issues/314 has been removed ?